### PR TITLE
Add fr.sgued.bodev

### PR DESCRIPTION
**Pull request by sgued**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly. Bodev is a developper toolbox. It contains a variety of tools that can be used for every day development tasks (conversions, CBOR debugging, Regex testing, etc...)
- [x] The domain used for the application ID is [controlled by the application developer(s)][appid-domain] and the [application id guidelines][appid] are followed (I already have published a verified application with the sgued.fr domain)
- [x] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [x] I have [built][build] and tested the submission locally.
  I get some lint errors I do not understand:
  - I don't understand the `ul` error, espcially given that other apps uses them.
  - I don't understand the external screenshot url either.
  - [It looks like the unnecessary xdg-config will need an exemption](https://github.com/flathub-infra/flatpak-builder-lint/pull/430)
```json
{ev.json
    "errors": [
        "appstream-failed-validation",
        "finish-args-unnecessary-xdg-config-cosmic-ro-access",
        "appstream-external-screenshot-url"
    ],
    "appstream": [
        "E: fr.sgued.bodev:14: description-para-markup-invalid ul"
    ],
    "info": [
        "appstream-external-screenshot-url: Screenshots are not mirrored to https://dl.flathub.org/media",
        "appstream-failed-validation: Metainfo file /tmp/tmp53v7m67v/metainfo/fr.sgued.bodev.metainfo.xml has failed validation. Please see the errors in appstream block"
    ],
    "message": "Please consult the documentation at https://docs.flathub.org/docs/for-app-authors/linter"
}
```

- [x] I am the author of the project.

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: fr.sgued.bodev.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "bodev",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:ro"
```
Comment (bbhtt) [inline fr.sgued.bodev.json:16]: this needs an exception https://github.com/flathub-infra/flatpak-builder-lint/blob/da1bd191e450b326276ebde60e9051568eda4525/flatpak_builder_lint/staticfiles/exceptions.json#L3841-L3842 can you open a PR?

**Comment by bbhtt** [comment]
>         "E: fr.sgued.bodev:14: description-para-markup-invalid ul"

You have to fix the metainfo file description https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#description

**Comment by bbhtt** [comment]
you can start the build by commenting in a few hours when the exception is up.

**Comment by sgued** [comment]
All should be fixed now.

**Comment by bbhtt** [comment]
bot, build fr.sgued.bodev

File: fr.sgued.bodev.json (hunk 1 above)
Comment (bbhtt) [inline fr.sgued.bodev.json:16]: you added an exception for rw but have `ro` here. I think this should be rw instead like all the other cosmic apps.

Comment (bbhtt) [reply fr.sgued.bodev.json:16]: See https://github.com/flathub-infra/flatpak-builder-lint/pull/534

Comment (sgued) [reply fr.sgued.bodev.json]: Right I forgot to update that. Thanks

**Comment by bbhtt** [comment]
bot, build fr.sgued.bodev

**Comment by bbhtt** [comment]
The text does not fit in the bubbles, not sure if that's intended but not a blocker.

image

**Pull request closed without merging**